exec/vector/src/main/java/org/apache/drill/exec/vector/accessor/ScalarReader.java

```diff
@@ -86,4 +87,10 @@
   LocalDate getDate();
   LocalTime getTime();
   Instant getTimestamp();
+
+  /**
+   * Return the value of the object using the extended type.
+   * @return
```

Comment:
Please move add description to the return to avoid warnings in the IDE (just move upper line to `@return`).